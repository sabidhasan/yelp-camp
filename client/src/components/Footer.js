import React from 'react'

const Footer = () => {
  return (
    <footer>
      <h2>Copyright YelpCamp 2018</h2>

      <h3>YelpCamp was created as part of <a href="#">The Web Developer Bootcamp </a>.
       This website has no affiliation with Yelp, Inc. in any way</h3>

      <h3>Made with <span className='footer__red'>‚ù§</span> in Edmonton, Canada</h3>

      <div className='social-media'>
        <a href='#'><i class="fab fa-twitter social-media__twitter"></i></a>
        <a href='#'><i class="fab fa-github-square social-media__github"></i></a>
        <a href='#'><i class="fab fa-stack-overflow social-media__stackoverflow"></i></a>
        <a href='#'><i class="fab fa-linkedin social-media__linkedin"></i></a>
      </div>
    </footer>
  )
}

export default Footer
