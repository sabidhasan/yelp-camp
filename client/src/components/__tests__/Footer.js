import React from 'react'
import toJson from 'enzyme-to-json'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Footer from '../Footer'

configure({ adapter: new Adapter() });

describe('<Footer />', () => {
  let component;

  beforeEach(() => {
    component = shallow(<Footer />)
  })

  it('Component should be defined', () => {
    expect(component).toBeDefined();
  });

  it('Footer component snapshot matches what is expected', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
