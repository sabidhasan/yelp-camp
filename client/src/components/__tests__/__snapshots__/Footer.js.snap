// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Footer component snapshot matches what is expected 1`] = `
<footer
  className="Footer flex-center"
>
  <h2
    className="Footer__text"
  >
    YelpCamp 2018
  </h2>
  <p
    role="contentinfo"
  >
    1234 Main St NW, Edmonton, AB T6J 1K1
    <br />
    <span
      aria-label="Phone Number"
      className="bold"
    >
      Phone
    </span>
    : 1 800 555 5555
    <br />
    <span
      aria-label="Email Address"
      className="bold"
    >
      Email
    </span>
    : contact@yelpcamp.com
  </p>
  <h2
    className="Footer__about"
  >
    This website has no affiliation with Yelp, Inc. in any way.
  </h2>
  <h2
    className="Footer__about"
  >
    Made with 
    <span
      aria-label="love"
      role="img"
    >
      ðŸ’š
    </span>
    and 
    <span
      aria-label="coffee"
      role="img"
    >
      â˜•
    </span>
     in Edmonton, Canada
  </h2>
  <div
    className="Footer__social"
  >
    <a
      aria-label="Twitter Link"
      className="Footer__social-link"
      href="http://twitter.com/abid786"
      id="Footer__social-twitter"
    >
      <i
        aria-labelledby="Footer__social-twitter"
        className="fab fa-twitter Footer__twitter"
      />
    </a>
    <a
      aria-label="Github Link"
      className="Footer__social-link"
      href="https://github.com/sabidhasan#"
      id="Footer__social-github"
    >
      <i
        aria-labelledby="Footer__social-github"
        className="fab fa-github-square Footer__github"
      />
    </a>
    <a
      aria-label="Stack Overflow Link"
      className="Footer__social-link"
      href="https://stackoverflow.com/users/7509481/abid-hasan"
      id="Footer__social-stack"
    >
      <i
        aria-labelledby="Footer__social-stack"
        className="fab fa-stack-overflow Footer__stackoverflow"
      />
    </a>
    <a
      aria-label="LinkedIn Link"
      className="Footer__social-link"
      href="https://www.linkedin.com/in/abidhasan/"
      id="Footer__social-linkedin"
    >
      <i
        aria-labelledby="Footer__social-linkedin"
        className="fab fa-linkedin Footer__linkedin"
      />
    </a>
  </div>
</footer>
`;
